Templates.sln

```diff
@@ -577,6 +577,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TimerTrigger-FSharp", "Time
 		Templates\TimerTrigger-FSharp\run.fsx = Templates\TimerTrigger-FSharp\run.fsx
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SnedGrid-Timer-CSharp", "SnedGrid-Timer-CSharp", "{E0D727E3-729D-419E-BF5D-2162846A69B3}"
```

Comment:
Sned :)

Also, recommend renaming the folder to match the new name "ScheduledMail-CSharp". Basically apply the rename throughout.